src/SourceBuild/content/repo-projects/fsharp.proj

```diff
@@ -8,6 +8,7 @@
 
     <!-- Restore and Build actions are already passed in by the root script. -->
     <BuildActions>$(FlagParameterPrefix)pack $(FlagParameterPrefix)publish</BuildActions>
+    <BuildActions Condition="'$(Sign)' == 'true'">$(BuildActions) $(FlagParameterPrefix)sign</BuildActions>
```

Comment:
Why did you add the `DotNetBuildSourceOnly` condition only to aspnetcore.proj?

Reply:
This was missed on my part. Fixed in https://github.com/dotnet/sdk/pull/44855/commits/4fed5a3acbedb056a3a1c1a82976edf5f7d09506